Assignment2/Salvador Gonzalez Garcia/graph.rdf

```diff
@@ -0,0 +1,40 @@
+<rdf:RDF
+xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+xmlns:class="http://www.ontologies.org/ontologies/class#"
+xmlns:sensor="http://www.ontologies.org/ontologies/sensor#"
+xmlns:computer="http://www.ontologies.org/ontologies/computer#"
+xmlns:measurement="http://www.ontologies.org/ontologies/measurement#"
+xmlns:user="http://www.ontologies.org/ontologies/user#"
+xmlns="http://www.oeg-upm.net/ontologies/class#"
+xml:base="http://www.oeg-upm.net/ontologies/class">
+	
+	<rdf:Property rdf:about="http://www.ontologies.org/ontologies/class#includes"/>
+	<rdf:Property rdf:about="http://www.ontologies.org/ontologies/sensor#hasMeasurement"/>
+	<rdf:Property rdf:about="http://www.ontologies.org/ontologies/measurement#hasTemperature"/>
+	<rdf:Property rdf:about="http://www.ontologies.org/ontologies/measurement#atTime"/>	
+	<rdf:Property rdf:about="http://www.ontologies.org/ontologies/computer#hasOwner"/>
+	<rdf:Property rdf:about="http://www.ontologies.org/ontologies/user#hasName"/>
+
+	<rdf:Description rdf:about="#Class01">
+		<class:includes rdf:resource="#Sensor029"/>
+		<includes rdf:resource="#Computer101"/>	
+	</rdf:Description>
+
+	<rdf:Description rdf:about="#Sensor029">
+		<sensor:hasMeasurement rdf:resource="Measurement8401"/>
+	</rdf:Description>
+
+	<rdf:Description rdf:about="#Computer101">
+		<computer:hasOwner rdf:resource="User10A"/>
+	</rdf:Description>
+
+	<rdf:Description rdf:about="#Measurement8401">
+		<measurement:hasTemperature rdf:Integer="24"/>		
```

Comment:
is rdf:Integer a valid property? You should better use a XSD type to your value.